src/components/reusable/notification/notification.ts

```diff
@@ -0,0 +1,107 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import NotificationScss from './notification.scss';
+import '../card';
+
+/**
+ * Notification
+ * @slot overflow-menu-slot - Slot for menu.
+ * @slot status-tag-slot - Slot for status tag.
+ * @slot unnamed - Slot for notification content.
+ * @fires on-notification-click - Emit event when click on notification
+ */
+
+@customElement('kyn-notification')
+export class Notification extends LitElement {
+  static override styles = NotificationScss;
+
+  /** Notification Title (Required). */
+  @property({ type: String })
+  notificationTitle = '';
+
+  /** Notification subtitle. */
+  @property({ type: String })
+  notificationSubtitle = '';
+
+  /** Notification description (Required). */
+  @property({ type: String })
+  description = '';
+
+  /** Timestamp of notification. */
+  @property({ type: String })
+  timeStamp = '';
+
+  /** Card href link */
+  @property({ type: String })
+  href = '';
+
+  override render() {
+    return html`
+      <kyn-card type="clickable" href=${this.href} target="_blank">
+        <div class="notification-wrapper">
+          <div class="notification-title-wrap">
+            <div class="notification-head">
+              <!-- Title -->
+              <h1 class="notification-title">${this.notificationTitle}</h1>
```

Comment:
Probably should not use any heading tag internally here, as it might break the heading hierarchy of the overall page it is used on. If it must be a heading tag, we might need to use a `<slot>` so a dev can pick the correct heading level.